### ISSUE TITLE
enhancement: a11y rule "no-noninteractive-element-interactions" fails with a warning. 

### ISSUE THREAD

**Issue by FatumaA** [open-sauced/app#4137]
### Describe the bug

[jsx-a11y eslint plugin](https://www.npmjs.com/package/eslint-plugin-jsx-a11y) was recently added to the project and this rule - jsx-a11y/no-noninteractive-element-interactions - fails with a warning.

This issue offers more context: https://github.com/open-sauced/app/issues/2264

### Steps to reproduce

The following components and files are causing the rule to fail:

1. components/molecules/HighlightInput/highlight-input-form.tsx

**Comment by Dun-sin** [comment]
.take
